src/H5C.c

```diff
@@ -33,6 +33,8 @@
  *
  *    Code Changes:
  *
+ *     - Remove extra functionality in H5C__flush_single_entry()?
```

Comment:
Please remove this comment and the one below, they have been removed multiple times and John's changes keep including them.